rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/FileMetadata.java

```diff
@@ -247,9 +248,9 @@ void deleteFiles(Cache<String, FileMetadata> cache)
     }
   }
 
-  public int getWeight(Configuration conf)
+  // Returns the current downloaded fileSize in KB
+  public int getWeight()
```

Comment:
same, should we add KB here as well?

Reply:
Also, should we get rid of 'getCurrentFileSize()'?

Reply:
storing weigh in KB is internal detail and not needed to be added to method name.

getCurrentFileSize is being used by tests, are you suggesting to change those tests?